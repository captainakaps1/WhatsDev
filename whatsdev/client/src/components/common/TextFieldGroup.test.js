import TextFieldGroup from "./TextFieldGroup";
import { render } from "@testing-library/react";

describe("Component Test - InputGroup", () => {
  const data = {
    name: "",
    placeholder: "",
    value: "",
    error: "",
    onChange: () => {},
    info: "",
    type: "",
    disabled: "",
  };

  it("matches snapshot", () => {
    const { asFragment } = render(<TextFieldGroup {...data} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders without crashing", async () => {
    const { findByTestId } = render(<TextFieldGroup {...data} />);
    expect(await findByTestId("Text-Field-Group-div")).toBeDefined();
  });
});
