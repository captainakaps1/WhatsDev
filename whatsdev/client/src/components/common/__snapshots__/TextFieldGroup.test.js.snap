// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Test - InputGroup matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="form-group"
    data-testid="Text-Field-Group-div"
  >
    <input
      class="form-control form-control-lg"
      name=""
      placeholder=""
      type=""
      value=""
    />
  </div>
</DocumentFragment>
`;
